### PR TITLE
feat: override item identifier validation when ff is enabled

### PR DESCRIPTION
**Pull request by bartlomiejmarszal**
This will override item identifier validation base on FEATURE_FLAG_UNIQUE_NUMERIC_QTI_IDENTIFIER

### CODE REVIEW

File: model/FeatureFlag/UniqueNumericQtiIdentifierClientConfig.php [hunk 1]
```diff
@@ -27,14 +27,15 @@
 
 class UniqueNumericQtiIdentifierClientConfig implements FeatureFlagConfigHandlerInterface
 {
+    public const QTI_ID_PATTERN = '/^\\d{9}$/';
```
Comment (Karol-Stelmaczonek) [inline model/FeatureFlag/UniqueNumericQtiIdentifierClientConfig.php:30]: Is that correct pattern with `\\d` ? Should it be `\d` ?

**Review by Karol-Stelmaczonek — APPROVED** [review]
- [x] New code is covered by tests (if applicable)
- [ ] Tests are running successfully (old and new ones) on my local machine (if applicable)
- [x] New code is respecting code style rules
- [x] New code is respecting best practices
- [ ] New code is not subject to concurrency issues (if applicable)
- [ ] Feature is working correctly on my local machine (if applicable)
- [x] Acceptance criteria are respected
- [x] Pull request title and description are meaningful

File: model/FeatureFlag/UniqueNumericQtiIdentifierClientConfig.php (hunk 1 above)
Comment (tikhanovichA) [reply model/FeatureFlag/UniqueNumericQtiIdentifierClientConfig.php:30]: Should we update it with Normalize string?

Comment (bartlomiejmarszal) [reply model/FeatureFlag/UniqueNumericQtiIdentifierClientConfig.php:30]: It should work. And we should keep it as regexp that will accept 9 digits. It is `UniqueNumericQtiIdentifier` validation.

**Review by tikhanovichA — APPROVED** [review]

**Pull request merged**